src/commands/actions/arguments/owner-nonce.ts

```diff
@@ -4,12 +4,12 @@ export default {
   options: {
     type: Number,
     required: true,
-    help: 'The nonce of the owner within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool'
+    help: 'The validator registration nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.'
```

Comment:
"The validator owner nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool."